package.json

```diff
@@ -66,8 +66,8 @@
     "webpack": "2.6.1"
   },
   "scripts": {
-    "test": "jest",
-    "test-integration": "jest tests_integration",
+    "test": "jest --forceExit",
```

Comment:
Can that be provided in the jest config instread?

Reply:
It seems it's just a [CLI option](https://facebook.github.io/jest/docs/en/cli.html#forceexit), but I'm still trying to mock the `get-stream` to avoid using Promise so as to prevent hanging, or is there something like `getStream.sync()`?

Reply:
OK, I removed those CLI options and mocked `get-stream`.